addons/api/mirage/config.js

```diff
@@ -859,6 +859,26 @@ function routes() {
     return appTokens.create(appTokenAttrs);
   });
 
+  this.post(
+    '/app-tokens/:idMethod',
+    function ({ appTokens }, { params: { idMethod } }) {
+      const attrs = this.normalizedRequestAttrs();
+      const id = idMethod.split(':')[0];
+      const method = idMethod.split(':')[1];
+      const appToken = appTokens.find(id);
+      let updatedAttrs = {};
+
+      if (method === 'revoke') {
+        updatedAttrs = {
+          version: attrs.version,
+          status: 'revoked',
```

Comment:
should we use the status var defined in the app tokens model instead of a string?

Reply:
Great suggestion. Let me get that fixed real quick like.